a11y-dialog.js

```diff
@@ -294,6 +294,10 @@ A11yDialog.prototype._fire = function (type, event) {
  * @param {Event} event
  */
 A11yDialog.prototype._bindKeypress = function (event) {
+  // This is an escape hatch in case there are nested dialogs, so the keypresses
+  // are only reacted to for the most recent one
+  if (!this.dialog.contains(document.activeElement)) return
```

Comment:
So beautifully simple! I mean compare this to the over-engineered blog post solutions at #80, my mind was on fire ahaha desperately trying to find patch!